### DISCUSSION TITLE
CAD application does not show line weights

### DISCUSSION THREAD

**Discussion by MarceloSanchez-Aerial** [mozman/ezdxf#490] [Q&A]
Hi, when using the script of ezdxf/examples/default_lineweights.py . All the lines have the same weight in the world and in the layout, only the difference in weight can be seen when exporting as pdf (image attached) how can I add line weight in the world view and in the layout?

![image](https://user-images.githubusercontent.com/84915678/124747462-3bcb6680-df22-11eb-910d-b77654dceaff.png)

**Comment by MarceloSanchez-Aerial** [comment]
I am using the free version of ZWCAD 2021

**Comment by mozman** [comment]
Most CAD applications require you to activate the line width display.

BricsCAD:
![image](https://user-images.githubusercontent.com/4230966/124749217-3d962980-df24-11eb-886f-499907d551d1.png)
